bun.yaml

```diff
@@ -86,6 +84,6 @@ test:
         bun --version
         bun init --yes
         bun run index.ts | grep "Hello via Bun!"
-        # TODO: Omit this test for now until upstream cut a new release (>= 1.1.44) with the fix and let's see if it will pass
+        # TODO: Omit this test for now until upstream cut a new release (>= 1.1.45) with the fix and let's see if it will pass
```

Comment:
this test is still running into the following issue:
```bash
2025/01/17 11:28:27 WARN /usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/14/../../../../include/c++/14/span:286: reference std::span<const unsigned char>::operator[](size_type) const [_Type = const unsigned char, _Extent = 18446744073709551615]: Assertion '__idx < size()' failed.
2025/01/17 11:28:27 WARN Aborted (core dumped)
```
this same is passing with upstream binaries.